base/src/main/java/io/spine/util/Exceptions.java

```diff
@@ -59,6 +61,7 @@ private Exceptions() {
      * @return nothing ever
      * @throws UnsupportedOperationException always
      */
+    @CanIgnoreReturnValue
```

Comment:
That's odd. IDEA gives a warning if the returned exception is not thrown:
```java
...
Exceptions.unsupported(...);   // the warning is emitted.

...
throw Exceptions.unsupported(...); // It's fine.
```
I don't know if `throw` counts as a usage of a returning value, but to my mind it should.

Therefore I wouldn't annotate methods like this one with `@CanIgnoreReturnValue`.